tests/tests/definition-list/definition-list.html

```diff
@@ -16,22 +15,18 @@ <h1>
         <dt>term 1</dt>
         <dd>Definition 1</dd>
         <dt>term 2</dt>
-        <dd>
```

Comment:
I believe the first 3 cases *are* according to spec (or at least, in sync with what Sphinx is producing).